_sass/_style.scss

```diff
@@ -326,6 +325,7 @@ a {
     margin: 0 auto 102.4px;
   }
   .inside-page-content {
+    max-width: 538px; /* 576 - 51.2 - 51.2 */
```

Comment:
The math here doesn't seem right. Did you mean to write something else in the comment?